README.md

```diff
@@ -1 +1,36 @@
-# hw01
+# hw01 - My Bio
+Hi, this is Zewei (Whiskey) Liao, a third-year Sociology Major and an international student from Shenzhen, China, a city right next to Hong Kong. 
+
+## The Origin of "Whiskey"
+Zewei is my original name in Chinese but since a lot of people (i.e. non-Chinese speakers) are having a hard time pronouncing it (and messing it up really badly into a word that sounds both hilariously and awkwardly inappropriate in Chinese -- see the bottom of this section), I usually just go by **Whiskey**, a nickname I picked for myself in primary school when I was crazy about a Japanese anime series in which all the antagonists, who are basically all gang members, are given code names based on alcoholic beverages. Yes, there are a wide variety of them, which include:
+- Gin
+- Vodka
+- Vermouth
+- Tequila
+- and basically all sorts of Whiskey (except for "Whiskey" itself), including:
+  - Bourbon
+  - Scotch
+  - Irish
+  - Rye
+
+Since none of them is actually "Whiskey," and the first syllable of "Whiskey" sounds very close to the last character of my name ("Wei" from "Zewei") both in Chinese and in English, I assumed my membership of the group with this coolest name.
+
+Check this [wiki page](https://www.wikiwand.com/en/List_of_Case_Closed_characters#Black_Organization) for a more comprehensive list of my fellows! 
+
+<details>
+  <summary> <i>Click here to see what "Zewei" sounds like in Chinese when it's mispronounced: </i> </summary>
+  Ziwei, which is what "Zewei" sounds like when mispronounced, means masturbation in Chinese
+  
+</details>
+
+## Some other things about me
+- I'm a decade-long Lakers fan :basketball: :trophy:
+- I also love Barcelona but I really have no idea what they are doing these days... :soccer: :man_facepalming:
```

Comment:
Me as well. Was sweating when Messi threatened to leave 😅

Reply:
> Me as well. Was sweating when Messi threatened to leave 😅

lmaoooo ikr

Reply:
> Me as well. Was sweating when Messi threatened to leave 😅

but wow fellow Barce fans here!!!